File: Curso Logica de Programação Neri Neitzke/Algoritmos em Pascal/pascal/tabuada_para.pas
// Prof Neri Aldoir Neitzke         - ULBRA
// www.INFORMATICON.com.br - videoaulasneri@gmail.com
{Problema: Ler uma variável de numero inteiro e mostrar a tabuada desse número. }
program tabuada_para;
var numero,i: integer;
begin
    write('Digite o numero : ');
    readln(numero);
    for i := 1 to 9 do
        writeln(numero  ,' x ',i,' = ',(numero * i));
    writeln('..................Final do Sistema..................... ');
    readln();
end.

File: Curso Logica de Programação Neri Neitzke/Algoritmos em Java/tabuada_para.java
// Prof Neri Aldoir Neitzke   ULBRA
// www.INFORMATICON.com.br - videoaulasneri@gmail.com
/*Problema: Ler uma variável de numero inteiro e mostrar a tabuada desse número.*/
import javax.swing.JOptionPane;
class tabuada_para
{
   public static void main(String args[])
   {
        int i, numero;
        String mensagem = "";
        numero = Integer.parseInt(JOptionPane.showInputDialog("Digite um numero.:"));
        for(i=1;i <= 9;i=i+1)      //i++ é igual a i=i+1;
            mensagem = mensagem + "\n"+numero + " x "+i+" = "+(numero * i);
        System.out.println(mensagem);
        JOptionPane.showMessageDialog(null, mensagem);

  }
}